Dockerfile.sdk

```diff
@@ -242,3 +242,4 @@ ENV NVIDIA_BUILD_ID=${NVIDIA_BUILD_ID}
 
 ENV PATH /workspace/install/bin:${PATH}
 ENV LD_LIBRARY_PATH /workspace/install/lib:${LD_LIBRARY_PATH}
+ENV LD_LIBRARY_PATH /opt/hpcx/ompi/lib:${LD_LIBRARY_PATH}
```

Comment:
Add a comment with the DLIS ticket Misha is working on over this line.

Reply:
Added comment 👍